subsys/testsuite/Kconfig

```diff
@@ -84,10 +85,20 @@ config COVERAGE_GCOV_HEAP_SIZE
 
 config COVERAGE_DUMP
 	bool "Dump coverage data on exit"
-	depends on COVERAGE_GCOV
 	help
 	  Dump collected coverage information to console on exit.
 
+config COVERAGE_DUMP_PATH_EXCLUSIVE
+	string "Only files matching this pattern are dumped in coverage data"
```

Comment:
naming is not ideal and confusing IMO, especially for non-native speakers. "exclusive" (read: "exclude") and yet this is a pattern to only _include_ files matching a given path?

```suggestion
config COVERAGE_DUMP_INCLUDE_PATTERN
	string "Only files matching this pattern are dumped in coverage data"
```

Reply:
> "exclusive" (read: "exclude") and yet this is a pattern to only include files matching a given path?

Yes, exclude any paths that don't match this pattern.
`INCLUDE_PATTERN` implies that this is in addition to some other path/pattern IMO.
<img width="714" height="393" alt="image" src="https://github.com/user-attachments/assets/3f210fbe-722e-4fa5-8491-20f2d135a245" />
I appreciate it may not be the clearest for non-native speakers, but isn't that what the help text etc are for?